test/helpers/add-pr-late-label.test.ts

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2021 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+  https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const mockList = jest.fn();
+jest.mock('../../src/octokit', () => ({
+  octokit: {
+    issues: {
+      addLabels: jest.fn()
+    },
+    pulls: {
+      list: mockList
+    }
+  }
+}));
```

Comment:
Imports should always come first in a file

Reply:
For some reason, when running with these specific jests after the imports, it doesnt seem to work at all.  Only way I could find to get it to work was to put it above? I think it has something to do with the difference between mocking '@action/octokit' and mocking the actual file '../../src/octokit'